### DISCUSSION TITLE
Octernship- Rustdesk task- Problem with set up

### DISCUSSION THREAD

**Discussion by JahnabDutta** [community/community#58163] [GitHub Education]
### Hi everyone,

I am trying to set up the codebase for the octernship task under rustdesk. There is a template which is supposed to run flutter with rust. The problem with the template is that I am unable to set up the project.

I am using Ubuntu 22.04 and I have flutter and rust installed on my pc. I am running the command `flutter run -d linux`. However I am getting the following error-
![image](https://github.com/community/community/assets/76475829/6e0e83b3-3507-440e-bc64-159851d2d53a)
The error seems to be generated from the following cause-
![image](https://github.com/community/community/assets/76475829/f3c75ca7-2b12-4292-9517-60fb7ceb51b8)

Can anyone help me set the project up please. Thanks!!

**Comment by dibenzepin** [comment]
can you try compiling just a regular rust project?

```sh
$ cd ~
$ cargo new this-is-a-test
$ cd this-is-a-test
$ cargo run
...
Hello, World!
```

**Reply by JahnabDutta** [reply]
Yeah that works fine for me.

**Reply by dibenzepin** [reply]
have you run the "Generating Code" section from the [flutter_rust_bridge](https://cjycode.com/flutter_rust_bridge/template.html) docs?

try running through that (i think you'll have to install [just](https://github.com/casey/just), and then run `just gen` to generate the bindings)
